stories/Form/Form.stories.js

```diff
@@ -264,10 +264,10 @@ stories.add("Readonly", withDocs(Readonly, withInfo()(ReadonlyComponent)));
 const ReadonlyNormalizzatoComponent = () => (
     <Form className="m-3">
         <FormGroup>
+            <Label for="staticEmail" style={{position:'static'}}>Email</Label>        
             <Input plaintext readOnly id="staticEmail">
                 email@example.com
             </Input>
-            <Label for="staticEmail">Email</Label>
```

Comment:
I think the label needs to appear below the email id as far as design is concerned of `Form` components. Seems that way in all other places.

Reply:
Hey @Diuda, It looks good to me. Have a  look at this [link](https://italia.github.io/bootstrap-italia/docs/form/select/). The labels are placed on the top.

Reply:
Just saw the link. I guess there's a conflict between the documentation on Bootstrap Italia page and how the labels have been structured in the current system then

Reply:
Yes, that could be the case.

Reply:
Yep, I confirm that. Things should be aligned to what you can see on Bootstrap Italia or on the [UI Kit](https://invis.io/VRR2TLB34TE) ([here](https://invis.io/D9R3VB52J8E#/291110316_It_Designers_UIkit_Components_Data_Entry_Input_Field) the page for input fields) that is the foundation for everything.